bapsflib/utils/__init__.py

```diff
@@ -11,10 +11,24 @@
 """
 Package of developer utilities.
 """
-__all__ = ['decorators', 'errors', 'warnings']
+__all__ = ['decorators', 'errors', 'warnings', 'NdarrayToXarray']
 
 from . import (decorators, errors, warnings)
 
 
-def foo(x):
-    return x
+def NdarrayToXarray(data, arr_name={}):
+    import xarray as xr
+    names = list(data.dtype.names)
+    names.remove('signal')
+    coords_dict = {}
+    for name in names:
+        if name == 'xyz':
+            coords_dict["x"] = ("shot_index", data[name][:, 0])
+            coords_dict["y"] = ("shot_index", data[name][:, 1])
+            coords_dict["z"] = ("shot_index", data[name][:, 2])
+
+        else:
+            coords_dict[name] = ("shot_index", data[name])
```

Comment:
```suggestion
        if name == 'xyz':
            coords_dict["x"] = ("shotnum", data[name][:, 0])
            coords_dict["y"] = ("shotnum", data[name][:, 1])
            coords_dict["z"] = ("shotnum", data[name][:, 2])

        else:
            coords_dict[name] = ("shotnum", data[name])
```

Since `bapsflib` utilizes the *shotnum* name in its conventions and that the plasmapy diagnostic will not care about the name, we should keep with the `bapsflib` convention.